docs/dx_tables.md

```diff
@@ -1,549 +1,74 @@
 
 # DX BUFR Tables
-@brief Description and format of DX BUFR tables for use with the library.
 @anchor dfbftab
 
```

Comment:
The \@brief is there for a reason.  If you go to the main documentation page (such as https://emc.ncep.noaa.gov/emc/pages/infrastructure/bufrlib_jeff/html/index.html which is the last time I downloaded and looked at the docs artifact from a developer CI run ;-), and if you then click on the "Related Pages" tab, you'll see the contents of the \@brief tag next to that page name.  Otherwise, if there's no \@brief in the .md file, then that part is blank with no description of what's actually in that page.  So it's a Doxygen thing, and I think it needs to be there.

Reply:
OK, but that also puts the brief at the top of the documentation page, which looks weird.

I'll leave it for now and if I figure out a better way to deal with it I'll circle back around to it...